apps/condo/domains/resident/tasks/sendBillingReceiptsAddedNotificationForOrganizationContextTask.js

```diff
@@ -18,8 +21,11 @@ const {
 } = require('@condo/domains/notification/constants/constants')
 const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
 const { BILLING_CONTEXT_SYNCHRONIZATION_DATE } = require('@condo/domains/resident/constants/constants')
-const logger = getLogger('sendNewBillingReceiptNotification')
 
+const logger = getLogger('sendNewBillingReceiptNotification')
+const BILLING_RECEIPT_FIELDS = 'id period toPay toPayDetails { charge } createdAt isPayable '
+    + 'context { id organization { id } integration { id currencyCode } } '
+    + 'account { id number } property { id address } category { id  nameNonLocalized }'
```

Comment:
If I understood correctly, you don't use all these fields. You are using only  id, period, toPay, toPayDetails, and re-requesting other fields

So better remove unused relations